app/views/searches/new.html.erb

```diff
@@ -1,15 +1,6 @@
-<form id="inline_searchform" class="form" autocomplete="off">
+<form id="inline_searchform" action=<%= search_query_path %> method="get" class="form" autocomplete="off">
```

Comment:
this one also needs quotation marks around the `action` parameter

Reply:
@jywarren aah sorry I got out of town and got busy with office work when came back, If its ok I will get back to it. Sorry again :disappointed:

Reply:
hi @Mujadded, sorry for your loss, and don't worry, you can complete this PR anytime at your convenience. Thanks so much for helping.

Reply:
If its ok I would close this pr and create a new one as the current branch is very far ahead

Reply:
created a new pr regarding this one https://github.com/publiclab/plots2/pull/2661